client/extensions/woocommerce/index.js

```diff
@@ -221,10 +221,23 @@ export default function() {
 	} );
 
 	// Add pages that use my-sites navigation instead
-	page( '/store/stats/:type/:unit', controller.siteSelection, controller.sites );
-	page( '/store/stats/:type/:unit/:site', siteSelection, navigation, StatsController );
+	page(
+		'/store/stats/:type/:unit',
+		controller.siteSelection,
+		controller.sites,
+		makeLayout,
+		clientRender
+	);
+	page(
+		'/store/stats/:type/:unit/:site',
+		siteSelection,
+		navigation,
```

Comment:
`navigation` renders to `#secondary` IIRC. The STR compliant counterpart is called `makeNavigation`. I wonder if we could/should solve that at codemod level.